### PR TITLE
fix: do not use cache in client-admin application

### LINKED ISSUE #39: Do not use "cache" fetch policy when making GraphQL requests in client-admin app
The admin app should always retrieve up to date data. 

While doing this issue, we should also:
- remove cache from client config
- remove offix

### PR DESCRIPTION
**Pull request by machi1990**
Fixes #39

##### Checklist

- [ ] `npm test` passes
- [ ] `npm run build` works
- [ ] tests are included
- [ ] documentation is changed or added

### CODE REVIEW

File: platform/client-admin/src/config/clientConfig.ts [hunk 1]
```diff
@@ -74,25 +71,22 @@ const splitLink = ApolloLink.split(
...
+    fetchPolicy: noCache,
+  },
+  mutate: {
+    fetchPolicy: noCache
   }
+}
+
+export const clientConfig = {
+  defaultOptions,
+  cache: new InMemoryCache(),
```
Comment (wtrocki) [inline platform/client-admin/src/config/clientConfig.ts:90]: Actually if we do not use cache maybe we simply do not pass cache at all.

**Comment by wtrocki** [comment]
I love this. Generally for the next days we will be able to try out URQL client in admin while still keeping offix and Apollo on volunteer side

File: platform/client-admin/src/config/clientConfig.ts (hunk 1 above)
Comment (machi1990) [reply platform/client-admin/src/config/clientConfig.ts:90]: Yes, my initial reaction but then `ts` complained.  Also seems like `apollo-client` will verify that a cache object is defined - worth a PR on their repo? WDYT?

**Comment by machi1990** [comment]
> I love this and how enclosed the client actually is now. Generally for the next days we will be able to try out URQL client in admin while still keeping offix and Apollo on volunteer side. Amazing work

By URQL client, you mean this client https://github.com/FormidableLabs/urql?

File: platform/client-admin/src/config/clientConfig.ts (hunk 1 above)
Comment (wtrocki) [reply platform/client-admin/src/config/clientConfig.ts:90]: Yes. I would do that as issue first and send PR if they like it

https://github.com/apollographql/apollo-client/blob/master/src/ApolloClient.ts#L42

**Comment by wtrocki** [comment]
Yep.. Do not want to bring it now but we will.l need to explore differences, developer experience etc.

File: platform/client-admin/src/config/clientConfig.ts (hunk 1 above)
Comment (machi1990) [reply platform/client-admin/src/config/clientConfig.ts:90]: What I am thinking is, if `no-cache` can be passed as default option, there is not need to make the https://github.com/apollographql/apollo-client/blob/8ee7648371fb3bac621af5c38fd98afb1f0bd7d7/src/ApolloClient.ts#L42

Comment (machi1990) [reply platform/client-admin/src/config/clientConfig.ts:90]: Ah seems like you beat me to the link ha ha

Comment (machi1990) [reply platform/client-admin/src/config/clientConfig.ts:90]: Also this condition will need changing I guess https://github.com/apollographql/apollo-client/blob/8ee7648371fb3bac621af5c38fd98afb1f0bd7d7/src/ApolloClient.ts#L140. 

But yes, good thinking. Let's open an issue for that first in their repo.

**Comment by machi1990** [comment]
Merged thanks. I'll open an issue later on apollo-client side. Thanks for the review.

**Pull request merged**